pkg/reconciler/revision/reconcile_resources.go

```diff
@@ -165,7 +159,7 @@ func (c *Reconciler) reconcilePA(ctx context.Context, rev *v1alpha1.Revision) er
 		want := pa.DeepCopy()
 		want.Spec = tmpl.Spec
 		if pa, err = c.ServingClientSet.AutoscalingV1alpha1().PodAutoscalers(pa.Namespace).Update(want); err != nil {
-			return err
+			return perrors.Wrapf(err, "failed to update PA %q", paName)
```

Comment:
here we can omit PA name, I guess (PA name == revision name) and that is the `key`.

Reply:
I'd keep these as is for now, to essentially keep identical information where it already was. It'll be part of my greater makeover to straighten that.